src/status_im/contexts/communities/overview/view.cljs

```diff
@@ -366,14 +367,30 @@
            ;; might have been removed.
            on-first-channel-height-changed}]]))))
 
-(defn- community-card-page-view
+(defn- community-fetching-placeholder
   [id]
+  (let [fetching? (rf/sub [:communities/fetching-community id])]
+    [rn/view
+     {:style               style/fetching-placeholder
+      :accessibility-label (if fetching?
+                             :fetching-community-overview
+                             :failed-to-fetch-community-overview)}
+     [not-implemented/not-implemented
+      [rn/text
+       (if fetching?
+         "Fetching community..."
+         "Failed to fetch community")]]]))
+
+(defn- community-card-page-view
+  [community-id]
```

Comment:
It's confusing to see `community-id` and `id` refer to the same thing in the same function. Could you just use one or the other?

Reply:
cause `id` can be `nil`, I was caught by it
I changed it back to id and removed the redefinition of id